Source/driver/math.H

```diff
@@ -0,0 +1,14 @@
+#ifndef CASTRO_MATH_H
+#define CASTRO_MATH_H
+
+AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
```

Comment:
Let's drop back to `AMREX_INLINE`, I think it was a mistake that I let `AMREX_FORCE_INLINE` proliferate.

Reply:
done